src/main/java/sirotkina/sjournal/utils/DatabaseUtils.java

```diff
@@ -0,0 +1,61 @@
+package sirotkina.sjournal.utils;
+
+import org.flywaydb.core.Flyway;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DatabaseUtils {
+    private static Connection connection;
+
+    static {
+        Runtime.getRuntime().addShutdownHook(new Thread(DatabaseUtils::closeConnection));
+    }
+
+    public static Connection getConnection() {
+        Properties properties = readProperties();
```

Comment:
create private static Properties dbProps in DatabaseUtils and read them once, during class loading